src/main/java/pl/edu/mimuw/Lecturer.java

```diff
@@ -0,0 +1,46 @@
+package pl.edu.mimuw;
+
+import java.util.HashSet;
+
+public class Lecturer extends Person {
+	private String academicTitle;
+	private HashSet<Publication> publications;
```

Comment:
trzymamy raczej definiując tym pola przez typ interfejsu dla kolekcji to znaczy
`Set<Publication>`